resources/assets/coffee/react/contest/voting/entry.coffee

```diff
@@ -24,12 +24,20 @@ class Contest.Voting.Entry extends React.Component
     if @props.contest.show_votes
       votePercentage = _.round((@props.entry.results.votes / @props.totalVotes)*100, 2)
       relativeVotePercentage = _.round((@props.entry.results.votes / @props.winnerVotes)*100, 2)
+      rank = @props.rank + 1
 
     selected = _.includes @props.selected, @props.entry.id
 
     div className: "contest-voting-list__row#{if selected && !@props.contest.show_votes then ' contest-voting-list__row--selected' else ''}",
+      if @props.contest.show_votes
+        div className: 'contest-voting-list__rank',
+          if rank < 4
+            div className: "contest-voting-list__trophy contest-voting-list__trophy--#{rank}",
```

Comment:
There's new `parentClass` props for `Icon` component which kind of simplifies this thing.